umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/provisioning.md

```diff
@@ -1,8 +1,47 @@
 ---
 title: Provisioning
 menuTitle: Provisioning
-description:
+description: |
+  Discover how to provision both the Data and the Device & Container
+  Infrastructures using the Management Console.
 weight: 1000
 edition: community
-draft: true
+draft: false
 ---
+
+The Management Console simplifies the deployment of the
+[Data Infrastructure](/docs/architecture/data-infrastructure/) on any existing
+system, and also guides you through the provisioning of the
+[Device & Container Infrastructure](/docs/architecture/device--container-infrastructure/).
+
+## When should I use it?
+
+Utilize the Management Console for initial deployment of the United
+Manufacturing Hub. It supports provisioning on both centralized servers and
+edge devices, offering flexibility in deployment options.
+
+## What can I do with it?
+
+Provision the United Manufacturing Hub on any existing system, and link it to
+the Management Console. The Management Console will then be able to manage the
+system and its resources.
+
+## How can I use it?
+
+To start, add a new instance in the Management Console. The Console provides a
+command that you execute on your target system, automating the installation of
+the United Manufacturing Hub. For detailed system requirements and
+compatibility, consult the [Installation](/docs/getstarted/installation/) guide.
+
+## What are the limitations?
+
+Current limitations include the necessity for pre-configured systems for
```

Comment:
What do you mean with a pre-configured system? It will allow also installing the UMH (maybe need to set the correct env variables)

Reply:
But maybe write that on al ater stage we will add the option to configure a full flatcar instance down to networking

Reply:
Pre-configured means that the operating system needs to already be installed, since the script will only install Companion (+ UMH, if the env var is set)